src/julia.h

```diff
@@ -1520,7 +1520,7 @@ typedef struct _jl_task_t {
 } jl_task_t;
 
 JL_DLLEXPORT jl_task_t *jl_new_task(jl_function_t *start, size_t ssize);
-JL_DLLEXPORT void jl_switchto(jl_task_t *t);
+JL_DLLEXPORT void jl_switchto(jl_task_t **pt);
```

Comment:
Yes. It also destroys its argument, which it pretty surprising for a typical API (effectively, this is now marked `std::move`). We haven't had a stable C API in the past however.